scene/gui/control.h

```diff
@@ -51,11 +51,9 @@ class Control : public CanvasItem {
 	OBJ_CATEGORY("GUI Nodes");
 
 public:
-	enum AnchorType {
-		ANCHOR_BEGIN,
-		ANCHOR_END,
-		ANCHOR_CENTER,
-	};
+	static const float ANCHOR_BEGIN = 0.0f;
```

Comment:
Getting an error with MSVC because of this: `error C2864: 'Control::ANCHOR_BEGIN': a static data member with an in-class initializer must have non-volatile const integral type`